backend/init_data/skills/dingtalk-knowledge/provider.py

```
@@ -0,0 +1,650 @@
+            if not self.auth_token:
+                raise RuntimeError("No authentication token available")
+
+            # Build curl command for upload
+            upload_curl_cmd = (
+                f"curl -s -X POST "
+                f'-H "Authorization: Bearer {self.auth_token}" '
+                f'-F "file=@{save_path}" '
+                f'"{upload_url}"'
+            )
+
+            logger.info(f"[DownloadDingTalkDocumentTool] Uploading to Wegent")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)